### ISSUE TITLE
[Feature Request] Update unpkg with react-grid-layout 0.14.0

### ISSUE THREAD

**Issue by ajabri** [STRML/react-grid-layout#495]
Awesome library :)
Could you publish react-grid-layout 0.14.0 to unpkg?

**Comment by STRML** [comment]
Unpkg, just mirrors npm. That said, 0.14.0 was not published to npm - it is now.

**Comment by ajabri** [comment]
Sorry, I was looking for the 'dist' directory in the unpkg version, and couldn't find it. Is https://cdn.rawgit.com/STRML/react-grid-layout/0.14.0/dist/react-grid-layout.min.js the only source of the repo minified? Thanks!

**Comment by STRML** [comment]
Yes - we don't publish the `dist/` folder to npm because it's not
technically part of the package, so there's no reason to load it.

**Comment by JohnDDuncanIII** [comment]
I apologize for bumping an ancient ticket, but I was wondering if it would eventually be possible to upload `dist/` to the npm version (which is then mirrored on https://unpkg.com/)?

---

> A module usable in a <script> tag is included here. It uses a UMD shim and excludes React, so it must be otherwise available in your application, either via RequireJS or on window.React

I wanted to test out some custom `react-grid-layout` functionality in a single `index.html` file for a non-production side project, but sadly, https://raw.githubusercontent.com/STRML/react-grid-layout/master/dist/react-grid-layout.min.js serves the file with header `Content-Type: text/plain; charset=utf-8` which causes

> Refused to execute script from 'https://raw.githubusercontent.com/STRML/react-grid-layout/master/dist/react-grid-layout.min.js' because its MIME type ('text/plain') is not executable, and strict MIME type checking is enabled.

when added to `index.html` as an import.

---

https://rawgit.com/STRML/react-grid-layout/master/dist/react-grid-layout.min.js works well, but the rawgit homepage now states

> GitHub repositories that served content through RawGit within the last month will continue to be served until at least October of 2019. URLs for other repositories are no longer being served.

> If you're currently using RawGit, please stop using it as soon as you can.

As far as I know of, the only other primary cdn which serves js script bundles is https://unpkg.com/.

---

tl;dr is that, in `index.html`:

- `<script crossorigin src="https://raw.githubusercontent.com/STRML/react-grid-layout/master/dist/react-grid-layout.min.js"></script>`
	- does not work since it is served over `Content-Type: text/plain; charset=utf-8`

- `<script crossorigin src="https://rawgit.com/STRML/react-grid-layout/master/dist/react-grid-layout.min.js"></script>`
	- works but is being taken down soon

- `<script crossorigin src="https://unpkg.com/react-grid-layout@latest/dist/react-grid-layout.min.js"></script>`
	- would be cool if this worked

**Comment by JohnDDuncanIII** [comment]
As an aside, I'm not sure if this is possible to fix, but it seems confusing that the `unpkg` mirror is currently serving what seems like an out of date https://unpkg.com/browse/react-grid-layout-2@latest/ as well as https://unpkg.com/browse/react-grid-layout@latest/.
